code/wdb/datastripe.q

```diff
@@ -57,25 +58,30 @@ if[.ds.datastripe;.proc.addinitlist[(`initdatastripe;`)]];
 \d .ds
 
 upserttopartition:{[dir;tablename;keycol;enumdata;nextp]
-        /- function takes a (dir)ectory handle, tablename as a symbol
-        /- column to key table on, an enumerated table and a timestamp.
-        /- partitions the data on the keycol and upserts it to the given dir
-        /- get unique sym from table
-        s:first raze value'[?[enumdata;();1b;enlist[keycol]!enlist keycol]];
-        /- get process specific taildir location
-        dir:` sv dir,.proc.procname,`$ string .wdb.currentpartition;
-        /- get symbol enumeration
-        partitionint:`$string (where s=value [`.]keycol)0;
-        /- create directory location for selected partition
-        directory:` sv .Q.par[dir;partitionint;tablename],`;
-        .lg.o[`save;"Saving ",string[s]," data from ",string[tablename]," table to partition ",string[partitionint],". Table contains ",string[count enumdata]," rows."];
-        .lg.o[`save;"Saving data down to ",string[directory]];
-        /- upsert select data matched on partition to specific directory
-        .[upsert;
-          (directory;enumdata);
-          {[e] .lg.e[`upserttopartition;"Failed to save table to disk : ",e];'e}
-        ];
-        };
+    /- function takes a (dir)ectory handle, tablename as a symbol
+    /- column to key table on, an enumerated table and a timestamp.
+    /- partitions the data on the keycol and upserts it to the given dir
+    
+    /- get unique sym from table
+    s:first raze value'[?[enumdata;();1b;enlist[keycol]!enlist keycol]];
+
+    /- get process specific taildir location
+    dir:` sv dir,.proc.procname,`$ string .wdb.currentpartition;
+
+    /- get symbol enumeration
+    partitionint:`$string (where s=value [`.]keycol)0;
+
+    /- create directory location for selected partition
+    directory:` sv .Q.par[dir;partitionint;tablename],`;
+    .lg.o[`save;"Saving ",string[s]," data from ",string[tablename]," table to partition ",string[partitionint],". Table contains ",string[count enumdata]," rows."];
+    .lg.o[`save;"Saving data down to ",string[directory]];
+
+    /- upsert select data matched on partition to specific directory
+    .[upsert;
+        (directory;enumdata);
+        {[e] .lg.e[`upserttopartition;"Failed to save table to disk : ",e];'e}
+    ];
+    };
```

Comment:
is there any actual functional change in this section or is it just layout changes? layout changes should typically be kept out of functional change PRs (can make a separate PR for layout changes if needed) as it makes it difficult to review what has actually changed (if anything) in this section.

Reply:
Just layout changes here - format was incongruent with rest of script, and difficult to read. I can revert this change & make a separate PR for it later, as you've suggested?